### PR TITLE
Update list & select function

### PR DESCRIPTION
**Pull request by hozongsien**
Completed port for list and select command:
- testcases have been updated
- a handful of testcase have been commented out awaiting other command ports.
    - Clear
    - Delete

### CODE REVIEW

**Comment by JunWei96** [comment]
Can make the PR to `Set basic functions for trivia` branch instead of master branch?

**Comment by JunWei96** [comment]
You branched out from `update-existing-trivia-functions` branch? haha I am still working on that branch tho, you should branch out `set-basic-functions-for-trivia` branch.

File: src/test/java/systemtests/SelectCommandSystemTest.java [hunk 1]
```diff
@@ -96,10 +99,11 @@ public void select() {
         /* Case: mixed case command word -> rejected */
         assertCommandFailure("SeLeCt 1", MESSAGE_UNKNOWN_COMMAND);
 
+        // TODO: when delete command is implemented
         /* Case: select from empty address book -> rejected */
-        deleteAllPersons();
-        assertCommandFailure(SelectCommand.COMMAND_WORD + " " + INDEX_FIRST_PERSON.getOneBased(),
-                MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+//        deleteAllPersons();
```
Comment (hozongsien) [inline src/test/java/systemtests/SelectCommandSystemTest.java]: Delete command not implemented

**Comment by JunWei96** [comment]
The travis test is failing because of some style error. You can run travis test locally by running the command as state in the post below:
https://github.com/orgs/CS2103-AY1819S1-W12-3/teams/developers/discussions/3

**Pull request merged**